ui/lib/core/addon/components/replication-header.js

```diff
@@ -9,8 +9,20 @@
  */
 
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import layout from '../templates/components/replication-header';
 
 export default Component.extend({
   layout,
+  data: null,
+  isSecondary: computed('data', function() {
+    let data = this.data;
+    let dr = this.data.dr;
```

Comment:
if you want to, you can shorthand this here with:

```
let dr = this.data && this.data.dr
if !(dr) {
 return false
}
return dr.mode === 'secondary'
```

also, what's `data.rm.mode`? do you need to verify that it's a dr cluster again, or isn't that implied if `this.data.dr` exists? 🤔

Reply:
Nice!  I need to know because I believe this.data.dr will return something, all the values will just be undefined.  Dr is always there, same with Performance.

Reply:
Actually, working through this I need the two laid variables separate like this because I use data and dr in the conditional.